tfm_ns_import.yaml

```diff
@@ -52,6 +52,10 @@
                 "src": "install/interface/src/tfm_crypto_ipc_api.c",
                 "dst": "platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/src/tfm_crypto_ipc_api.c"
             },
+            {
+                "src": "install/interface/src/tfm_firmware_update_ipc_api.c",
+                "dst": "platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/src/tfm_firmware_update_ipc_api.c"
+            },
```

Comment:
CI failed because the firmware update API is not available on all targets. Let me think about it.

Reply:
It's fine to add this file. On a target that doesn't support Firmware Update, compilation still works, and any attempt to call the Firmware Update API returns a runtime error which is good enough.

Because it's not supported on Musca S1, could you please move it to `ARM_MUSCA_B1` above?